include/ddprof_defs.hpp

```diff
@@ -13,6 +13,11 @@
 // Maximum depth for a single stack
 #define DD_MAX_STACK_DEPTH 512
 
+// Sample stack size must a multiple of 8 and strictly inferior to 2^16
+// The type is uint32 to be consistent with the perf_event interface
+// Check linux sources for a reference to this check
```

Comment:
Why not leave the other part of the comment ?
>Note that since maximum perf_event_hdr size is 2^16-1 and there are other
> data/headers in perf_event struct, actual maximum stack sample size returned
> as `size_stack` might be smaller then the requested size